File: Course_01/C Language/EstruturaRepetitiva/tabuada.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n, r, i;

    printf("Deseja a tabuada para qual valor? ");
    scanf("%d", &n);

    for (i = 1 ; i < 11 ; i++) {
        r = n * i;
        printf("%d x %d = %d\n", n, i, r);
    }
    return 0;
}

File: Course_01/Java Language/Estrutura repetitiva/tabuada.java
package Repetitivo;

import java.util.Locale;
import java.util.Scanner;

public class tabuada {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int i, n;
		
		System.out.print("Deseja a tabuada para qual valor? ");
		n = sc.nextInt();
		
		for (i = 1; i <= 10; i++) {
			System.out.println(n + " x " + i + " = " + i * n);
		}
		
		sc.close();
	}

}
